CODIGO/General.bas

```diff
@@ -1037,22 +1037,21 @@ End Function
 Sub Main()
 
 On Error GoTo Main_Err
-    Set FormParser = New clsCursor
-    Call FormParser.Init
     
     Call LoadConfig
-    
+    Call SetLanguageApplication
```

Comment:
Did you run the client with `stop in all errors`

Usually if we move the order things are initialized in `main` something breaks.

Reply:
yes I tested this, but without this change the loading takes to long to show up, because loading time with the anti cheat are longer, there is long time between time that the anticheat  window disappear  and the argetum loading show up and it seems like a bug